src/content/job/director_ecosystem.md

```diff
@@ -1,31 +1,33 @@
 ---
-title: Community Manager, Full Time, Remote
+title: Director, Ecosystem
 ---
-Posted on August 18th, 2023. 
+Posted on August 18th, 2023.
```

Comment:
@beecastanon Do you mean, remove this?  Yes, we can do that. (Please just clarify that this is the "timestamp" to which you refer.)

Reply:
@WrathfulSpatula thanks! "Posted on August 18th, 2023. Updated on September 19th, 2023." is the text that I would imagine may be confusing to folks.